docs/030_user-guide/010_pepr-cli.md

```diff
@@ -12,7 +12,7 @@ Initialize a new Pepr Module.
 - `--name <string>` - Set the name of the new module.
 - `--skip-post-init` - Skip npm install, git init, and VSCode launch.
 - `--errorBehavior <audit|ignore|reject>` - Set an errorBehavior.
-
+- `--uuid [string]` - Unique identifier for your module..
```

Comment:
```suggestion
- `--uuid [string]` - Unique identifier for your module. Must be 36 characters or fewer and a [kubernetes acceptable name](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/)
```

Reply:
This is the output from the init help, this does not correct to what the command says

```ts
> npx ts-node src/cli.ts init --help
Usage: cli init [options]

Initialize a new Pepr Module

Options:
  --confirm                              Skip verification prompt when creating a new module.
  --description <string>                 Explain the purpose of the new module.
  --name <string>                        Set the name of the new module.
  --skip-post-init                       Skip npm install, git init, and VSCode launch.
  --errorBehavior <audit|ignore|reject>  Set an errorBehavior.
  --uuid [string]                        Unique identifier for your module.
  -h, --help                             display help for command
```

Reply:
resolved as we are describing length constraints in the cli and the same message in the docs.